### PR TITLE
Full form bug/icon added, closes issue #233

### PR DESCRIPTION
**Pull request by praptisharma28**
Corrected the full form and added a Linkedin icon in footer to increase the open source contributors and followers on linkedin!

### CODE REVIEW

File: src/components/Footer.jsx
Comment (A91y) [inline src/components/Footer.jsx:1]: Certain elements in footer are missing. Please fix those.
Original: 
Screenshot 2024-01-07 160935

Footer you created:
Screenshot 2024-01-07 161148

**Review by A91y — CHANGES_REQUESTED** [review]

File: src/components/Footer.jsx
Comment (praptisharma28) [reply src/components/Footer.jsx:1]: Done @A91y , don't know how I missed it.

**Review by codecShivam — CHANGES_REQUESTED** [review]
image

###### @praptisharma28 Ensure that the text regarding 'fosscu' and its open-source components is included, and once that's done, everything lgtm :+1:

**Comment by praptisharma28** [comment]
Done @codecShivam

**Comment by Mr-Sunglasses** [comment]
@praptisharma28 Please open a corresponding issue and link this PR. to that issue it will help us in tracking the progress during releases.

**Comment by praptisharma28** [comment]
> @praptisharma28 Please open a corresponding issue and link this PR. to that issue it will help us in tracking the progress during releases.

https://github.com/FOSS-Community/website-fossc/issues/233 @Mr-Sunglasses

**Pull request closed without merging**